assume/common/outputs.py

```diff
@@ -58,6 +58,9 @@ def __init__(
         self.write_dfs: dict = defaultdict(list)
 
         if self.db is not None:
+            # TODO: #84 check if simulation_id is already in db
```

Comment:
I would not want to documente this twice - as you already created an issue for this